Gemfile

```diff
@@ -89,6 +89,7 @@ group(:omnibus_package) do
   gem "cookstyle"
   gem "winrm-fs"
   gem "winrm-elevated"
+  gem "cucumber-core", "< 2.0.0"
```

Comment:
I had to add this because junkins was failing without it:
```
:ruby_22, :ruby_23, :mri, :mri_18, :mri_19, :mri_20, :mri_21, :mri_22, :mri_23, :rbx]}])
                          D | 2016-09-07T22:15:00-07:00 | Fetching https://github.com/berkshelf/berkshelf-api.git
                          D | 2016-09-07T22:15:03-07:00 | Fetching gem metadata from https://rubygems.org/
                          D | 2016-09-07T22:15:04-07:00 | Fetching version metadata from https://rubygems.org/
                          D | 2016-09-07T22:15:04-07:00 | Fetching dependency metadata from https://rubygems.org/
                          D | 2016-09-07T22:15:50-07:00 | Resolving dependencies.................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................
                          D | 2016-09-07T22:15:50-07:00 | Bundler could not find compatible versions for gem "cucumber-core":
                          D | 2016-09-07T22:15:50-07:00 |   In Gemfile:
                          D | 2016-09-07T22:15:50-07:00 |     cucumber-core (= 2.0.0)
                          D | 2016-09-07T22:15:50-07:00 | 
                          D | 2016-09-07T22:15:50-07:00 |     aruba (>= 0.10.0) was resolved to 0.10.0, which depends on
                          D | 2016-09-07T22:15:50-07:00 |       cucumber (>= 1.3.19) was resolved to 2.1.0, which depends on
                          D | 2016-09-07T22:15:50-07:00 |         cucumber-core (~> 1.3.0)
                          D | 2016-09-07T22:15:50-07:00 | 
                          D | 2016-09-07T22:15:50-07:00 |     foodcritic (= 7.1.0) was resolved to 7.1.0, which depends on
                          D | 2016-09-07T22:15:50-07:00 |       cucumber-core (>= 1.3)
                          D | 2016-09-07T22:15:50-07:00 | Bundler could not find compatible versions for gem "gherkin":
                          D | 2016-09-07T22:15:50-07:00 |   In Gemfile:
                          D | 2016-09-07T22:15:50-07:00 |     gherkin (= 4.0.0)
                          D | 2016-09-07T22:15:50-07:00 | 
                          D | 2016-09-07T22:15:50-07:00 |     aruba (>= 0.10.0) was resolved to 0.10.0, which depends on
                          D | 2016-09-07T22:15:50-07:00 |       cucumber (>= 1.3.19) was resolved to 1.3.19, which depends on
                          D | 2016-09-07T22:15:50-07:00 |         gherkin (~> 2.12)
                          D | 2016-09-07T22:15:50-07:00 | 
                          D | 2016-09-07T22:15:50-07:00 |     cucumber-core (= 2.0.0) was resolved to 2.0.0, which depends on
                          D | 2016-09-07T22:15:50-07:00 |       gherkin (~> 4.0)
[Builder: chef-dk-appbundle] I | 2016-09-07T22:15:50-07:00 | Lock down the berkshelf gem: 53.1205s
[Builder: chef-dk-appbundle] I | 2016-09-07T22:15:50-07:00 | Build chef-dk-appbundle: 66.2518s
The following shell command exited with status 6:
```